src/views/other-reason-disclaimer.njk

```diff
@@ -38,15 +38,16 @@
           <li>the directors live or were travelling overseas</li>
           <li>another director is responsible for preparing the accounts</li>
         </ul>
-        {{ 
-          govukButton({
-            text: "Continue with appeal",
-            href: "#",
-            classes: "govuk-link govuk-!-margin-top-5 govuk-link--interrupt"
-          }) 
-        }}
-        <p>
-          <a href="#" class="govuk-link">Save and sign out</a>
-        </p>
+        <form method="post">
+          {{ 
+            govukButton({
+              text: "Continue with appeal",
+              classes: "govuk-link govuk-!-margin-top-5 govuk-link--interrupt"
+            }) 
+          }}
+          <p>
+            <a href="#" class="govuk-link">Save and sign out</a>
```

Comment:
we should probably get rid of this button and not include it on any other page even though the prototype has it, since it's not something that's going to be implemented in our first release

Reply:
we'll have to run this past our lovely BA/QA